### PR TITLE
3383cab: Implementation of slider with interesting builds section

### PR DESCRIPTION
**Pull request by Logovas1994**

### CODE REVIEW

**Comment by nastyakuzusha** [comment]
Header is missing for the Carousel

**Comment by nastyakuzusha** [comment]
Screenshot 2022-12-22 at 13 09 40
border is hidden for hover on the corner cards

File: front/src/components/nft-carousel/NftCarousel.tsx [hunk 1]
```diff
@@ -0,0 +1,18 @@
+import { NFTCard } from "../../schema/NFTCatalogSchema";
+import { useStyles } from "./NftCarouselStyles";
+import {GCCarousel} from "../../gc-components/carousel/GCCarousel";
+
+interface NftCarouselProps {
+  data: NFTCard;
+}
+
+const NftCarousel = ({data}: NftCarouselProps) => {
```
Comment (nastyakuzusha) [inline front/src/components/nft-carousel/NftCarousel.tsx]: why do you throw data here in props but dont use it?

Comment (nastyakuzusha) [reply front/src/components/nft-carousel/NftCarousel.tsx]: why do you even need this component?

File: front/src/components/nft-carousel/NftCarouselStyles.ts [hunk 2]
```diff
@@ -0,0 +1,5 @@
+import { createStyles } from "@mantine/core";
+
+export const useStyles = createStyles((theme) => ({
```
Comment (nastyakuzusha) [inline front/src/components/nft-carousel/NftCarouselStyles.ts:3]: why do you need this file?

**Comment by nastyakuzusha** [comment]
Screen Recording 2022-12-22 at 13.16.11

https://user-images.githubusercontent.com/14147976/209133109-cfa49a12-f85d-4e13-b91d-801579be8462.mov


There is no possibility to scroll for the slider for some of the devices

**Comment by nastyakuzusha** [comment]
Screenshot 2022-12-22 at 13 19 07
same device, cards are too far away from each other

**Comment by nastyakuzusha** [comment]
Screenshot 2022-12-22 at 13 20 17
big-width phone should have the same gallery width as the one only card you show

File: front/src/gc-components/carousel/GCCarouselSlide.tsx [hunk 3]
```diff
@@ -0,0 +1,15 @@
+import { Carousel, CarouselProps } from "@mantine/carousel";
+import React, { ReactNode } from "react";
+
+interface GCCarouselSlideProps extends CarouselProps {
+    children: ReactNode;
+}
+
+export const GCCarouselSlide = ({children, ...props}: GCCarouselSlideProps) => {
```
Comment (nastyakuzusha) [inline front/src/gc-components/carousel/GCCarouselSlide.tsx]: you dont use this component anywhere

**Comment by nastyakuzusha** [comment]
Screenshot 2022-12-22 at 14 02 19
its kinda not a loop for high resolution screens

**Review by nastyakuzusha — APPROVED** [review]

File: package.json [hunk 4]
```diff
@@ -0,0 +1,6 @@
+{
+  "dependencies": {
```
Comment (AbrosimovVladislav) [inline package.json]: почему это файл не во фронте? и где мантайн карусель внутри пэкэдж жсон в папке фронт? ветка не запускается, таких пул реквестов быть не должно

**Comment by AbrosimovVladislav** [comment]
Снимок экрана 2022-12-26 в 21 17 03

**Review by nastyakuzusha — APPROVED** [review]

File: front/src/pages/nft/[id].tsx [hunk 5]
```diff
@@ -1,20 +1,25 @@
...
 import NftCardInformation from "../../components/nft-card-information/NftCardInformation";
 import Catalog from "../../components/catalog/Catalog";
-import {Product} from "../../schema/NFTCatalogSchema";
+import { Product } from "../../schema/NFTCatalogSchema";
 import GunPartCard from "../../components/gun-part-card/GunPartCard";
+import NftCarousel from "../../components/nft-carousel/NftCarousel";
 
 type SingleNFTPageProps = {};
 
 const SingleNFTPage = (props: SingleNFTPageProps) => {
```
Comment (AbrosimovVladislav) [inline front/src/pages/nft/[id].tsx:19]: если пропсы здесь не юзаем, то надо убрать их из передачи в параметры

**Review by AbrosimovVladislav — APPROVED** [review]

**Pull request merged**